pkg/compact/compact.go

```diff
@@ -345,8 +480,20 @@ func (cg *Group) Labels() map[string]string {
 }
 
 // Compact plans and runs a single compaction against the group. The compacted result
-// is uploaded into the bucket the blocks were retrived from.
-func (cg *Group) Compact(ctx context.Context, comp tsdb.Compactor) (id ulid.ULID, err error) {
+// is uploaded into the bucket the blocks were retrieved from.
+func (cg *Group) Compact(ctx context.Context, comp tsdb.Compactor) (ulid.ULID, error) {
+	begin := time.Now()
+
+	id, err := cg.compact(ctx, comp)
+	if err != nil {
+		cg.metrics.compactionFailures.Inc()
+	}
+	cg.metrics.compactionDuration.Observe(time.Since(begin).Seconds())
+
```

Comment:
Also a metric for total compactions? I know this is kind of included in the the duration metric as a generated one. But it's usually unintuitive for people to use this one and a dedicated one is easier.

Reply:
so it is why I did not include it.. It is already there in Duration